README.md

```diff
@@ -298,12 +277,42 @@ objectives:
    values to find bad values. If `Good` is defined then `Bad` should not be set.
 
   - *Total* represents the query used for gathering data from metric sources
-  that is used as the denominator. Received data is used to compare objectives
-  (threshold) values to find total number of metrics.
+   that is used as the denominator. Received data is used to compare objectives
+   (threshold) values to find total number of metrics.
 
-###### Notes (Ratio metrics)
+An example of an SLO where SLI is inlined:
 
-If a service level indicator has `ratioMetrics` defined, the following maths can
+```yaml
+apiVersion: openslo/v0.1.0-beta
+kind: SLO
+metadata:
+  name: foo-slo
+  displayName: Foo SLO
+spec:
+  indicator:
+    kind: SLI
```

Comment:
If it's inline, would it have the `kind` set? Or is it only needed when it's an ObjectReference?

Reply:
Having kind set would align with existing convention for all inline usages but seems a little bit verbose indeed.

Reply:
Do you have any examples of its use? I tried to find some and failed miserably with my Google fu

Reply:
Guess, you could consider it is implied to be of kind `SLI` when it's in the `spec.indicator`-block. I think it other places we do the same though 🤔

Reply:
Thanks for the feedback. I'll remove `kind: SLI` since it's under `spec.indicator`.